reference/artifacts/types/overview/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: single
+title:  "Types of Artifacts"
+sidebar:
+  nav: reference
+---
+
+While an artifact can reference any remote, deployable resource, there are
+certain types of artifacts that we have first-class support for in the form of:
+
+* Parsing events from other services. For example, reading pub/sub messages
+  from GCR images.
+
+* Credentials for downloading artifacts. For example, a GitHub access token to
+  read repository contents.
+
+* Integrations with stages that require certain types of artifacts. For
+  example, App Engine can only deploy a Docker image as a custom runtime, never
+  an AMI.
+
+The pages in the section serve to document the format Spinnaker expects these
```

Comment:
"The pages in *this* section describe the format Spinnaker expects for these artifacts, to make custom integrations easier."